modules/compactor/config.go

```diff
@@ -5,7 +5,8 @@ import (
 	"fmt"
 	"time"
 
-	cortex_compactor "github.com/cortexproject/cortex/pkg/compactor"
+	tempo_compactor "github.com/grafana/tempo/pkg/compactor"
```

Comment:
remove alias. `compactor` is fine.

Reply:
Oh, I see, you moved it to `/pkg/compactor``. Let' s just move this code directly into `/modules/compactor`. It's only used by that module and we'd prefer keeping it isolated there to making a new module.

Generally we only put things in `/pkg` that will be shared by multiple components of Tempo.

Reply:
Sure thing, I've updated to keep it all in the `modules/`.  For the distributor PR also.